scripts/diff.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const cp = require('child_process');
+const { JSDOM } = require('jsdom');
+
+const BUILD_PATH = path.resolve(__dirname, '../out/index.html');
+const DIFF_PATH = path.resolve(__dirname, './htmldiff.pl');
+const OLD_PATH = path.resolve(os.tmpdir(), './old.html');
+const NEW_PATH = path.resolve(os.tmpdir(), './new.html');
+
+const last = cp.execSync('git show origin/gh-pages:index.html', {
```

Comment:
`origin` isn't necessarily the remote name; mine's `source` (`origin` is my personal fork)

Reply:
i'm not sure what to do. just `git show gh-pages:index.html` isn't enough.

Reply:
I came up with `execSync('git branch -a | grep "gh-pages"').toString().trim()`, which works great locally, but fails on netlify